web/packages/teleterm/src/ui/Search/pickers/ResultList.tsx

```diff
@@ -43,13 +42,19 @@ type ResultListProps<T> = {
   onPick(item: T): void;
   onBack(): void;
   render(item: T): { Component: ReactElement; key: string };
+  addWindowEventListener: AddWindowEventListener;
 };
 
 export function ResultList<T>(props: ResultListProps<T>) {
-  const { attempts, ExtraTopComponent, onPick, onBack } = props;
+  const {
+    attempts,
+    ExtraTopComponent,
+    onPick,
+    onBack,
+    addWindowEventListener,
+  } = props;
   const activeItemRef = useRef<HTMLDivElement>();
   const [activeItemIndex, setActiveItemIndex] = useState(0);
-  const { addWindowEventListener } = useSearchContext();
```

Comment:
Adding `useSearchContext` to `ResultList` broke the `ActionPicker` story because the story was not wrapping that component in the context. I refactored `ResultList` to take `addWindowEventListener` as a prop instead. It is a pretty much a self-contained component with only presentational logic so in the end I figured it might not make sense to contaminate it with a context.